DEVELOPER.md

```diff
@@ -11,46 +11,14 @@ Things to watch out for:
 - The goal is for Twinkle and Morebits to support the same [browsers for which MediaWiki provides Grade A support](https://www.mediawiki.org/wiki/Browser_compatibility). The Twinkle gadget on enwiki is configured so that we can use up to JavaScript version ES6. However, due to the MediaWiki minifier, we must not use keywords from ES2016 or later, such as async/await and RegEx /s flag. New functions from ES2016 or later, such as Array.includes() should be okay since these will not break the minifier.
 - Certain positional jQuery selectors like `:first`, `:last`, and `:eq` were [deprecated in jQuery version 3.4.0](https://blog.jquery.com/2019/04/10/jquery-3-4-0-released/) and should probably not be reintroduced. Instead, use methods like `.first()`, `.last()`, or `.eq()`.
 
-## Updating scripts on Wikipedia
+## Deploying
 
-There are two ways to upload Twinkle scripts to Wikipedia or another destination. You can do it with a [Perl script](#synchronization-using-syncpl) (recommended) or [manually](#manual-synchronization).
+There are two ways to upload Twinkle scripts to Wikipedia or another destination. 1) You can use the deploy script by creating a scripts/credentials.json file and then doing `npm run deploy` (see below section for more details). There are also some [on-wiki instructions](https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Articles_for_creation/Helper_script/Deploying). Or 2) you can visit https://gadget-deploy.toolforge.org/, log in, then click the deploy button.
```

Comment:
might be a bit confusing to refer to the AFCH's help page, even though steps are mostly same as for Twinkle